frontend/src/static/js/v_zoom.js

```diff
@@ -1,9 +1,23 @@
+window.toggleCommitBodyMessage = function toggleCommitBodyMessage(ele) {
```

Comment:
Why can't we use the existing `toggleNext` function? I believe some renaming is enough for this case. The logic is just the same.

Reply:
This isn't possible due to the way the HTML element is being arranged, and that `toggleNext` always grabs the parentNode of the element instead of the current node

Reply:
Actually, it can be done using scss. For example, if we use the current toggle method, the `active` class will be added to the `li` of `commit-message`. Then we can have a scss of :

```
.commit-message.active  {
    .body {
    ...
    }
}
.commit-message  {
    .body {
        display: none
    }
}
``` 
So, it will also work. 
But it is on you to decide which one is better. I am just giving an suggestion.

Reply:
Noted. Let's use toggleNext instead to try and minimize duplicate codes